ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.less

```diff
@@ -39,14 +39,6 @@
   }
 }
 
-@{customElementNamespace}result-row:not(.is-downloadable) {
```

Comment:
This `is-downloadable` is toggled here https://github.com/cassandrabailey293/ddf/blob/DDF-UI-115-thumbnail-display/ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.view.js#L113 can it be removed as well?

Reply:
done 👍